packages/ses/test/test-sloppy-globals-scope-terminator.js

```diff
@@ -0,0 +1,87 @@
+import test from 'ava';
+import { createSloppyGlobalsScopeTerminator } from '../src/sloppy-globals-scope-terminator.js';
+
+test('sloppyGlobalsScopeTerminator/get - always has properties but they are undefined', t => {
+  t.plan(4);
+
+  const globalObject = {};
+  const scopeTerminator = createSloppyGlobalsScopeTerminator(globalObject);
+
+  t.is(Reflect.has(scopeTerminator, 'eval'), true);
+  t.is(Reflect.get(scopeTerminator, 'eval'), undefined);
+
+  t.is(Reflect.has(scopeTerminator, 'xyz'), true);
+  t.is(Reflect.get(scopeTerminator, 'xyz'), undefined);
+});
+
+test('sloppyGlobalsScopeTerminator/set - sets happen on the globalObject', t => {
+  t.plan(6);
+
+  const globalObject = {};
+  const scopeTerminator = createSloppyGlobalsScopeTerminator(globalObject);
+
+  t.is(Reflect.set(scopeTerminator, 'eval', 42), true);
+  t.is(Reflect.get(scopeTerminator, 'eval'), undefined);
+  t.is(Reflect.get(globalObject, 'eval'), 42);
+
+  t.is(Reflect.set(scopeTerminator, 'xyz', 42), true);
+  t.is(Reflect.get(scopeTerminator, 'xyz'), undefined);
+  t.is(Reflect.get(globalObject, 'xyz'), 42);
+});
+
+test('sloppyGlobalsScopeTerminator/getPrototypeOf - has null prototype', t => {
+  t.plan(1);
+
+  const globalObject = {};
+  const scopeTerminator = createSloppyGlobalsScopeTerminator(globalObject);
+
+  t.is(Reflect.getPrototypeOf(scopeTerminator), null);
+});
+
+test('sloppyGlobalsScopeTerminator/getOwnPropertyDescriptor - always has start compartment properties but provides no prop desc', t => {
```

Comment:
The `getOwnPropertyDescriptor`'s trap use of `console.warn` seem to be there to debug an obscure issue. I'm not sure it's worth testing that `console.warn` actually gets called. @erights you added this console logging I believe, do we care about testing it?

Reply:
After discussing this in the meeting today, let's drop the `console.warn` part of this test.